crates/relayer/src/chain/cosmos/estimate.rs

```diff
@@ -175,3 +194,13 @@ fn can_recover_from_simulation_failure(e: &Error) -> bool {
         _ => false,
     }
 }
+
+
+fn get_error_text(e: &Error) -> String {
+    use crate::error::ErrorDetail::*;
+
+    match e.detail() {
+        GrpcStatus(detail) => detail.status.code().to_string(),
```

Comment:
this one is to prevent error_descriptions like `gRPC call `send_tx_simulate` failed with status: status: Unknown, message: "spendable balance 2950583uosmo is smaller than 3000001uosmo: insufficient funds: insufficient funds [osmosis-labs/osmosis/v22/x/txfees/keeper/feedecorator.go:294] With gas wanted: '300000000' and gas used: '529599' ", details: [], metadata: MetadataMap { headers: {"content-type": "application/grpc", "x-cosmos-block-height": "13728350"} }`, and to kinda group them by error message, not sure if that can be done in a better way, please let me know if so

Reply:
also I am not sure if there's a way to get the code of the underlying gRPC error as they are all `Unknown`, if there is, I guess it would be better to use this one (and its text description), but not sure if it is possible